src/login/components/DataPanel/EditMode.js

```diff
@@ -26,17 +38,12 @@ const EditMode = (props) => {
         </li>
       </nav>
       <div className="group-data">
-        <pre>{JSON.stringify(props.data, null, 2)}</pre>
+        <SingleGroup group={props.group} />
```

Comment:
I would like to keep this `SingleGroup`, in the EditMode function. Was there a reason to split the function? is the `SingleGroup` suposed to develop in the future?

Reply:
ok! let's keep it as it is!